torchaudio/csrc/iir_cuda.cu

```diff
@@ -0,0 +1,78 @@
+#include <torch/extension.h>
```

Comment:
`<torch/extension.h>` header for writing Python binding and it attempts to include Python header. Here `torch/torch.h` should do.

```suggestion
#include <torch/torch.h>
```